red/api/flow.js

```diff
@@ -29,7 +29,11 @@ module.exports = {
         var flow = redNodes.getFlow(id);
         if (flow) {
             log.audit({event: "flow.get",id:id},req);
-            res.json(flow);
+            if (settings.flowFilePretty) {
+                flowDef = JSON.stringify(flowDef,null,4);
+            }
+            res.setHeader('Content-Type', 'application/json');
+            res.send(flow);
```

Comment:
It depends what problem you're trying to solve. The core of the requirement is for the flow file on disk to be formatted, which doesn't require any changes to the api.